DSA/DSSession1/Stack/StackArrayTest.java

```diff
@@ -0,0 +1,92 @@
+package GET2018.DSA.DSSession1.Stack;
+
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import GET2018.DSA.DSSession1.Stack.StackArray.StackException;
+
+/**
+ * This class performs various tests on StackArray class methods
+ * 
+ * @author Mohit Sharma
+ *
+ */
+public class StackArrayTest {
```

Comment:
Please use "JUnit’s ExpectedException" in test classes and verify exception and messages with this approach.